libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -237,7 +237,6 @@ - (void)send:(SFRestRequest *)request delegate:(id<SFRestDelegate>)delegate shou
         [SFSDKCoreLogger i:[self class] format:@"No auth credentials found. Authenticating before sending request: %@", request.description];
         [[SFUserAccountManager sharedInstance] loginWithCompletion:^(SFOAuthInfo *authInfo, SFUserAccount *userAccount) {
             __strong typeof(weakSelf) strongSelf = weakSelf;
-            [SFUserAccountManager sharedInstance].currentUser = userAccount;
```

Comment:
What was that line for?

Reply:
SFUserAccountManager:loginwithCompletion should set it. No need to explicitly set it anymore.